base/setup/usetup/format.c

```diff
@@ -96,6 +96,8 @@ FormatPartition(
     if (!FileSystem || !FileSystem->FormatFunc)
         return STATUS_NOT_SUPPORTED;
 
+    CONSOLE_ClearTextXY(6, 10, 100);
```

Comment:
Could you add a C comment in the code, above this line, that says that you
```
Clear the 'Press ENTER to continue' message
```
? (this help the code reader to know what it is about, instead of losing time by doing a "git blame" to check which revision introduced this line and for which purpose).

Reply:
I have added:
    /* Remove "Setup will now format the partition. 
     * Press ENTER to continue." message */
Is it OK?